helpers/blog_helper.rb

```diff
@@ -3,15 +3,10 @@ module BlogHelper
   # Try to switch over to super.url_slug, if possible!
   def self.url_slug(post)
     # This is Split out for Testing Purposes.
-    return url_slug_from_wordpress_url(post&.wordpress_url) if post&.wordpress_url&.present?
+    return post.url_slug if post&.url_slug&.present?
     return url_slug_from_title(post&.title)
```

Comment:
My only suggestion is, is there a place where we don't need this—and it's always a `post.url_slug`?

Reply:
I just wanted to handle the case where a url slug isn't set yet; for old articles. This way I didn't need to write a migration